gtk4/src/auto/menu_button.rs

```diff
@@ -369,12 +473,16 @@ impl Default for MenuButton {
 ///
 /// [builder-pattern]: https://doc.rust-lang.org/1.0.0/style/ownership/builders.html
 pub struct MenuButtonBuilder {
+    #[cfg(any(feature = "v4_4", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v4_4")))]
+    always_show_arrow: Option<bool>,
     direction: Option<ArrowType>,
     has_frame: Option<bool>,
     icon_name: Option<String>,
     label: Option<String>,
     menu_model: Option<gio::MenuModel>,
     popover: Option<Popover>,
+    primary: Option<bool>,
```

Comment:
This lacks a since tag compared to `always_show_arrow`

Reply:
fixed